src/.editorconfig

```diff
@@ -0,0 +1,9 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
If you format your files using CoreFx' .editorconfig, it will fix all trailing white space issues:
https://github.com/dotnet/corefx/blob/master/.editorconfig